packages/amplication-prisma-db/prisma/migrations/20220524102539_create_only_new_project_hierarchy/migration.sql

```diff
@@ -0,0 +1,5 @@
+INSERT INTO "Project"("id", "workspaceId", "name") 
+SELECT "id", "workspaceId", CONCAT('project-', "id") FROM "App";
```

Comment:
it is better to the use app name for the project name instead of CONCAT('project-', "id")